llvm/test/Verifier/ifunc-macho.ll

```diff
@@ -0,0 +1,42 @@
+; RUN:  not llvm-as %s -o /dev/null 2>&1 | FileCheck %s
```

Comment:
This test is leftover from a different approach I took earlier in the history of this patch series. I don't think we actually need it.